### ISSUE TITLE
Can't find BasicPlayerController component documentation

### ISSUE THREAD

**Issue by lortschi** [threlte/threlte#947]
I see BasicPlayerController component implemented into Threlte/ Rapier <world> showcase visualisation. This component is doing a lot under the hood and has a few properties. Sadly I can't find any documentation about this neither nor on Threlte or Rapier.

**Comment by DefinitelyMaybe** [comment]
Someone can correct me if I'm wrong but it's something we won't be documenting within threlte as I think it's an [abstraction rapier made](https://rapier.rs/docs/user_guides/javascript/character_controller)

**Comment by lortschi** [comment]
> Someone can correct me if I'm wrong but it's something we won't be documenting within threlte as it's an [abstraction rapier made](https://rapier.rs/docs/user_guides/javascript/character_controller)

Thanks @DefinitelyMaybe  for picking this up. I see on Rapier's Charachter Controller completely different properties as on Threlte's. These from Threlte are there not mentioned:

 position?: Parameters<Vector3['set']> | undefined;
 height?: number | undefined;
 radius?: number | undefined;
 speed?: number | undefined;
 jumpStrength?: number | undefined;
 playerCollisionGroups?: CollisionGroupsBitMask | undefined;
 groundCollisionGroups?: CollisionGroupsBitMask | undefined;
 
Probably Rapier's is poor documented that's why I think would be great to have a richer one in Threlte. Also under Threlte's default props I see an possible events delegation, but can't figure out how to pass this to this partial component.

**Comment by DefinitelyMaybe** [comment]
wait no, I tell a lie, [BasicPlayerController](https://github.com/threlte/threlte/blob/main/packages/rapier/src/lib/recipes/BasicPlayerController.svelte) is a export we made. I'm not sure of the exact intentions but perhaps it is more of an example for folks to play around with, get started quickly and remix when needs change

In terms of character controllers a port of [ecctrl](https://github.com/pmndrs/ecctrl) would be something that I'd like to eventually support and document but it's not a current priority

**Comment by lortschi** [comment]
> wait no, I tell a lie, [BasicPlayerController](https://github.com/threlte/threlte/blob/main/packages/rapier/src/lib/recipes/BasicPlayerController.svelte) is a export we made. I'm not sure of the exact intentions but perhaps it is more of an example for folks to play around with, get started quickly and remix when needs change
> 
> In terms of character controllers a port of [ecctrl](https://github.com/pmndrs/ecctrl) would be something that I'd like to eventually support and document but it's not a current priority

Yes, eectrl, that looks more like it hehe. Of course, ecctrl looks very good in terms of settings. I hope that one day the port will be complete so that 3rd person games can be completed more quickly. Threlte is still very much behind R3F solutions but the future and performance belongs to Svelte / Threlte :) 

Thank you guys for your support on this project!

**Comment by DefinitelyMaybe** [comment]
☺️ making 3D web stuff easy is the goal. Happy for folks to use different tools, we all end up learning from one another 

I believe [camera-controls](https://threlte.xyz/docs/examples/camera/camera-controls) has a third person mode if that'd help 😊
